tests/trainer/test_trainer_cli.py

```diff
@@ -0,0 +1,87 @@
+import inspect
+from argparse import ArgumentParser, Namespace
+from unittest import mock
+
+import pytest
+
+import tests.models.utils as tutils
+from pytorch_lightning import Trainer
+
+
+@mock.patch('argparse.ArgumentParser.parse_args',
+            return_value=Namespace(**Trainer.default_attributes()))
+def test_default_args(tmpdir):
+    """Tests default argument parser for Trainer"""
+    tutils.reset_seed()
+
+    # logger file to get meta
+    logger = tutils.get_test_tube_logger(tmpdir, False)
+
+    parser = ArgumentParser(add_help=False)
+    args = parser.parse_args()
+    args.logger = logger
+
+    args.max_epochs = 5
+    trainer = Trainer.from_argparse_args(args)
+
+    assert isinstance(trainer, Trainer)
+    assert trainer.max_epochs == 5
+
+
+@pytest.mark.parametrize(
+    'cli_args',
+    [
+        ['--accumulate_grad_batches=22'],
+        ['--print_nan_grads=1', '--weights_save_path=./'],
+        []
+    ]
+)
```

Comment:
```suggestion
@pytest.mark.parametrize('cli_args', [
        ['--accumulate_grad_batches=22'],
        ['--print_nan_grads=1', '--weights_save_path=./'],
        []
])
```
just a bit shorter lol